templates/common/copilot-gpt-basic/appPackage/declarativeCopilot.json.tpl

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://graphdevxdata.blob.core.windows.net/data/schemas/CopilotGPTManifestSchema-1.0.json",
```

Comment:
Do we also need to include other fields like name, description?

Reply:
Yes, we need. But I'm unsure about what should be written here. I'll update the content based on what PM shares next.

Reply:
Based on the document Sébastien just shared, I think I know what to write now. Let me update this content.